libplatsupport/src/plat/quartz64/serial.c

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
I wonder, is this basically a copy of `src/plat/rockpro64/serial.c`? This could move to `src/mach` then to avoid code duplication.

Reply:
It is, except the UART addresses are different.